build.gradle

```diff
@@ -0,0 +1,47 @@
+
+plugins {
+    id 'java-gradle-plugin'
+    id 'com.gradle.plugin-publish' version '0.10.1' 
+    id 'maven-publish'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(group: 'org.cyclonedx', name: 'cyclonedx-core-java', version: '2.1.1') {
+        // gradle-api already includes an slf4j binding
+        exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
```

Comment:
Not sure why cyclonedx-core-java has an _implementation_ for slf4j in it.  Generally bad practice for libraries to include implementations of the logger.  Those should be bound by the runtime environment, i.e. gradle, in this case.

Reply:
Agreed. I think its inherited due to the spdx-tools dependency, which is a bit of a mess.